raiden/blockchain_events_handler.py

```diff
@@ -475,7 +475,7 @@ def handle_channel_batch_unlock(raiden: "RaidenService", event: Event):
         transaction_hash=transaction_hash,
         canonical_identifier=canonical_identifier,
         participant=args["participant"],
-        partner=args["partner"],
+        partner=partner,
```

Comment:
Partner here can be 2 things,  depending on which participant has the raiden.address, why this was using args["partner"] instead of this var?
Maybe the args["partner"] contemplates some other use cases that we're not seeing here.
I'll not mark this as a change for this PR cause maybe it's ok this change, but i only want to know why this change is needed.

Reply:
Youre right, and since we need to implement the unlock for lc, i will rollback this change and check the tests results.